src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/CSharpTagHelperCodeRenderer.cs

```diff
@@ -440,7 +440,7 @@ private void RenderEndTagHelpersScope()
                                                   _tagHelperContext.ScopeManagerEndMethodName);
         }
 
-        private void RenderTagOutput(string tagOutputMethodName)
+        private void RenderTagOutput(string tagOutputMethodName, bool isString = false)
```

Comment:
wouldn't need this parameter at all if the `Generate...` methods returned a consistent type